torchgeo/datamodules/mmflood.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""MMFlood datamodule."""
+
+from typing import Any, Literal
+
+import kornia.augmentation as K
+import torch
+from kornia.constants import DataKey, Resample
+
+from ..datasets import MMFlood
+from ..samplers import GridGeoSampler, RandomBatchGeoSampler
+from ..samplers.utils import _to_tuple
+from .geo import GeoDataModule
+
+
+class MMFloodDataModule(GeoDataModule):
+    """LightningDataModule implementation for the MMFlood dataset.
+
+    .. versionadded:: 0.7
+    """
+
+    # Computed over train set
+    mean = torch.tensor([0.1785585, 0.03574104, 168.45529])
+    median = torch.tensor([0.116051525, 0.025692634, 86.0])
+    std = torch.tensor([2.405442, 0.22719479, 242.74359])
```

Comment:
Do these work both `include_dem` True and False? Seems like the length should change